supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -64,32 +63,39 @@ Workaround: Rerun the deployment command, or use a local container image registr
 
 For guidance and the template text to use for Technology Preview features, see the xref:technology-preview-guidance[Technology Preview] section.
 
-=== Deprecated functionality
-Warn users about the following deprecation stages:
+[[deprecated-and-removed-features]]
+=== Deprecated and removed features
+
+Documenting the deprecation and removal stages of software features requires careful and precise communication.
+Highlight the following stages to users:
 
 * Plan to deprecate
 * Deprecate
 * Plan to remove
 * Remove
 
-If available, inform users of alternative capabilities and workarounds.
+When alternatives to or workarounds for deprecated features are available, clearly inform users about them.
+
+==== Referring to releases in deprecation and removal notices
+In general, avoid definitive statements about specific releases, release versions, or dates for deprecation or removal.
+When possible, use the phrase "is planned for a future release" because it accounts for the possibility of changes to the planned deprecation or removal timeline.
+
+If you must be specific about a release, use provisional language to reflect the fluid nature of development plans and to acknowledge the potential for plans to change.
+For example, if you must cite a specific version, rather than stating "<x> will be deprecated in version 4.16", use "It is currently planned for <x> to be deprecated in version 4.16".
+Alternatively, if you must cite a deprecation or removal timeline and you want to avoid citing a specific release number, use a phrase such as "<x> will be deprecated in the next release".
```

Comment:
Love this new rewrite! The only thing I'd suggest is to still leave wiggle room in here:

```suggestion
Alternatively, if you must cite a deprecation or removal timeline and you want to avoid citing a specific release number, use a phrase such as "<x> is planned to be deprecated in the next release".
```

Or "planned for deprecation" or whatever is the least awkward wording :smile: Or you could always align it more with the exact wording of line 84 and just replace the 4.16 with "the next release"

Reply:
Good edit, @bergerhoffer. I'll make the change.